stats/opentelemetry/internal/pluginoption.go

```diff
@@ -27,13 +27,12 @@ import (
 // intended to be added to applicable OpenTelemetry metrics recorded in the
 // OpenTelemetry instrumentation component.
 //
-// This API is experimental. In the future, we hope to stabilize and expose this
-// API to allow plugins to inject labels of their choosing into metrics
-// recorded.
+// In the future, we hope to stabilize and expose this API to allow plugins to
+// inject labels of their choosing into metrics recorded.
 type PluginOption interface {
 	// GetMetadata creates a MD with metadata exchange labels.
 	GetMetadata() metadata.MD
-	// GetLabels emits relevant labels from the metadata provided and the
-	// optional labels provided, alongside any relevant labels.
-	GetLabels(metadata.MD, map[string]string) map[string]string
+	// GetLabels emits relevant labels from the incoming RPC metadata, alongside
+	// any relevant labels.
```

Comment:
Delete "alongside" stuff since that refs the labels you used to pass in?

Reply:
Alongside is semantically referring to the local labels I emit from the Get call unconditionally (derived at plugin option creation time).